lib/test_prof/logging.rb

```diff
@@ -13,7 +13,7 @@ def log(level, msg)
     end
 
     def build_log_msg(level, msg)
-      colorize(level, "[TEST PROF #{level.to_s.upcase}] #{msg}")
+      colorize(level, "\n[TEST PROF #{level.to_s.upcase}] #{msg}")
```

Comment:
What's the reason for the change? I think, it's better to control newlines in-place and do not hide within a private method.

Reply:
Yep, you're right - it was just a workaround for default Minitest reporter, where progress line concatenates with EventProf message. I'll remove it.